src/lib/getDateFromNow.test.ts

```diff
@@ -0,0 +1,20 @@
+import getDateFromNow from "./getDatefromNow";
+import { describe, expect, test } from "vitest";
+
+describe("Test: getDateFromNow() without suffix", () => {
+  test("should return 20 years", () => {
+    const dateString = "1999-01-01";
+    const result = getDateFromNow(dateString, true);
+
+    expect(result).toBe("24 years");
+  });
+});
+
+describe("Test: getDateFromNow() with suffix", () => {
+  test("should return 20 years", () => {
+    const dateString = "1999-01-01";
+    const result = getDateFromNow(dateString);
+
+    expect(result).toBe("24 years ago");
```

Comment:
You could try something to this effect in order to always set back 20 years from today:
```javascript
const d = new Date();
// set date object back 20 years
d.setFullYear(d.getFullYear()-20);
// if d is Date object for November 3, 2002, dateString = "2002-11-03".
const dateString = d.toISOString().substring(0,10);

```
Full disclosure, I haven't tried that locally, so it may not work exactly as written.

Reply:
Implemented this and it worked 🍕. Thanks